node/_tools/test.ts

```diff
@@ -70,7 +74,7 @@ for await (const path of testPaths) {
         console.log(`Error: "${path}" failed`);
         console.log(
           "You can repeat only this test with the command:",
-          magenta(cmd.join(" ")),
+          magenta(`deno test -A node/_tools/test.ts -- ${path}`),
```

Comment:
This shows the message like: `You can repeat only this test with the command: deno test -A node/_tools/test.ts -- parallel/test-child-process-execfile.js`